iron-overlay-behavior.html

```diff
@@ -11,8 +11,8 @@
 <link rel="import" href="../polymer/polymer.html">
 <link rel="import" href="../iron-fit-behavior/iron-fit-behavior.html">
 <link rel="import" href="../iron-resizable-behavior/iron-resizable-behavior.html">
-<link rel="import" href="iron-overlay-backdrop.html">
 <link rel="import" href="iron-overlay-manager.html">
+<link rel="import" href="iron-overlay-backdrop.html">
```

Comment:
hmm, sure. My reasoning was that the backdrop needs the manager before but if I think about it, the backdrop actually imports the manager, so should be fine. I'll do that now.